### ISSUE TITLE
Why does the begining so shaking？

### ISSUE THREAD

**Issue by yuchenlichuck** [zychen-ustc/XDancer#7]
https://github.com/user-attachments/assets/47bcc6fa-cf5e-4f7d-a1f6-279c029ea3b5

I found all the results is so shaking and the init video is so bad

**Comment by yuchenlichuck** [comment]
https://github.com/user-attachments/assets/03660777-dcd3-4426-8882-0725555f5908

**Comment by yuchenlichuck** [comment]
https://github.com/user-attachments/assets/04416145-d5be-4128-a16c-b8eecaf1a00e
https://github.com/user-attachments/assets/d07144ab-7325-4b42-9996-230885010bfd
https://github.com/user-attachments/assets/d11abeaf-2762-4487-b9ec-ccc8ceaeea30
https://github.com/user-attachments/assets/144b9d0c-500f-4910-ba95-726723fddba7
https://github.com/user-attachments/assets/884217ff-7c02-497a-98f0-977466b8effc

**Comment by zychen-ustc** [comment]
The most possible reason is a misalignment for joint ids.

**Comment by yuchenlichuck** [comment]
I think I didn't change the code, is it the reason of train or inference?

**Comment by zychen-ustc** [comment]
Which pkl file are you using as the reference pose?

The pkl file is supposed to be extracted from this [file](https://github.com/zychen-ustc/XDancer/blob/main/XDancer-motion/mmpose_inference_pose.py).

**Comment by yuchenlichuck** [comment]
sure let me try again

**Comment by yuchenlichuck** [comment]
ash inference_motion.sh
/mnt/raid/yuchen/envs/xdancer_motion/lib/python3.10/site-packages/torch/cuda/__init__.py:65: FutureWarning: The pynvml package is deprecated. Please install nvidia-ml-py instead. If you did not install pynvml directly, please report this to the maintainers of the package that installed pynvml for you.
  import pynvml  # type: ignore[import]
Some weights of GPT2LMHeadModel were not initialized from the model checkpoint at GPT2-medium and are newly initialized because the shapes did not match:
- wpe.weight: found shape torch.Size([1024, 1024]) in the checkpoint and torch.Size([2560, 1024]) in the model instantiated
- wte.weight: found shape torch.Size([50257, 1024]) in the checkpoint and torch.Size([2560, 1024]) in the model instantiated
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
loading checkpoint from ./MultipartVQ.pth
loading transformer checkpoint from ./MusicGPT.pth
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.

I find this warning. is it because of this reason?

**Comment by yuchenlichuck** [comment]
okay okay I think I got reasonable results: 
Thank you for your support

https://github.com/user-attachments/assets/c8f6d32d-149b-44c4-8bdb-c69c8fcb3a32

**Comment by ReAklL** [comment]
> 好的好的，我觉得我得到了比较合理的结果： 谢谢你的支持
> 
>  mKR3.mp4

您好，请问最后是怎么解决您提出的这个问题的，然后如果我想用自己的一张照片和一段音乐，可以实现吗？它效果怎么样？求大佬回复
